Library/PhotosController.swift

```diff
@@ -70,6 +91,13 @@ class PhotosController: UICollectionViewController {
 
         return alertController
     }
+
+    func object(at indexPath: IndexPath) -> Photo {
```

Comment:
If this method always return a `Photo` why isnt it called `photo(at indexPath:)`?